bundle/ScheduledVisibility/Handler/ObjectState.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility\Handler;
+
+use Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException;
+use Ibexa\Contracts\Core\Repository\ObjectStateService;
+use Ibexa\Contracts\Core\Repository\Repository;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use Ibexa\Contracts\Core\Repository\Values\ObjectState\ObjectState as ObjectStateValue;
+use Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility\ScheduledVisibilityInterface;
+
+final class ObjectState implements ScheduledVisibilityInterface
+{
+    public function __construct(
+        private readonly Repository $repository,
+        private readonly ObjectStateService $objectStateService,
+        private readonly int $objectStateGroupId,
+        private readonly int $hiddenObjectStateId,
+        private readonly int $visibleObjectStateId,
+    ) {}
+
+    /**
+     * @throws NotFoundException
```

Comment:
All of these are superfluous here.

`@throws` should be used ONLY for exceptions that are thrown directly by the method in question.

Reply:
fixed in fa153b003652a49ea510b3b043c68e87d73d325f